cmd/catalyst-node/catalyst-node.go

```diff
@@ -424,18 +424,58 @@ func writeSerfConfig(config *agent.Config) (string, error) {
 }
 
 func startCatalystWebServer(httpAddr string, redirectPrefixes []string) {
-	http.Handle("/hls/", redirectHlsHandler(redirectPrefixes))
+	http.Handle("/", redirectHandler(redirectPrefixes))
 	glog.Infof("HTTP server listening on %s", httpAddr)
 	glog.Fatal(http.ListenAndServe(httpAddr, nil))
 }
 
 var getClosestNode = queryMistForClosestNode
 
-func redirectHlsHandler(redirectPrefixes []string) http.Handler {
+// return the best node available for a given stream. will return any node if nobody has the stream.
+func getBestNode(redirectPrefixes []string, playbackID, lat, lon string) (string, string, error) {
```

Comment:
Nit: I suggest to move this function under `redirectHandler()`. It's simpler to read when you follow the [Stepdown Rule](https://dzone.com/articles/the-stepdown-rule).